city_scrapers/spiders/chi_north_river_mental_health.py

```diff
@@ -0,0 +1,73 @@
+import re
+from dateutil.parser import parse
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiNorthRiverMentalHealthSpider(CityScrapersSpider):
+    name = "chi_north_river_mental_health"
+    agency = "North River Expanded Mental Health Services Program and Governing Commission"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.northriverexpandedmentalhealthservicescommission.org"]
+    start_urls = ["http://www.northriverexpandedmentalhealthservicescommission.org/minutes.html"]
+
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for item in response.css(".meetings"):
```

Comment:
The `.meetings` selector is mostly a placeholder for getting started, but usually sites don't have meetings organized under a "meetings" class. It looks like on the minutes page the right selector to go through each link would be `.wsite-not-footer a`.

You can iterate through that, and then pass that `item` variable to each `_parse` function